src/App.jsx

```diff
@@ -28,8 +28,24 @@ function App() {
       <p className="read-the-docs">
         Click on the Vite and React logos to learn more
       </p>
+        <ul>
+            <lis></lis>
+            <li>blue blaaaaa<li>
+                <ol>
+                    <li></li>
+                    <li></li>
+                    <li></li>
+                </ol>
+        </ul>
+        <div>
+            <form action="">
+                Naam:
```

Comment:
Label 'Naam:' lijkt onvolledig, mist mogelijk invoerveld of bijbehorend element.